frontend/src/components/SyncEditor/SyncEditor.tsx

```diff
@@ -652,12 +653,22 @@ export function SyncEditor(props: SyncEditorProps): JSX.Element {
     if (variant === 'toolbar') {
       height -= 36
     }
+    if (editorHasErrors) {
+      height -= 36
+    }
```

Comment:
Can we combine these two blocks to avoid repeating this hard-coded height adjustment?

Reply:
so, I forgot to adjust the editorHasErrors value-- it's meant to be the height of the error alert so that the editor isn't pushed down past the bottom of the window and for that alert is should be 75---it's not in a place I can dynamically determine th height